distribution/lib/Standard/Base/0.0.0-dev/src/Data/Locale.enso

```diff
@@ -6,308 +6,303 @@ polyglot java import java.util.Locale as JavaLocale
-new : Text -> Text | Nothing -> Text | Nothing -> Locale
-new language country=Nothing variant=Nothing =
-    country_text = country.if_nothing ""
-    variant_text = variant.if_nothing ""
-    java_locale = JavaLocale.new language country_text variant_text
-    from_java java_locale
+       > Example
+         Get the South Korean locale.
 
-## Returns the locale specified by the provided IETF BCP47 language tag string.
+             import Standard.Base.Data.Locale
 
-   ? Language Tag Syntax
-     If the specified language tag contains any ill-formed subtags, the first
-     such subtag and all following subtags are ignored.
+             example_locale = Locale.south_korea
+    south_korea : Locale
+    south_korea = Locale.from_language_tag "ko-KR"
 
-     The following conversions are performed:
-     - The language code "und" is mapped to language "".
-     - The language codes "he", "yi", and "id" are mapped to "iw", "ji", and
-       "in" respectively.
-     - The portion of a private use subtag prefixed by "lvariant", if any, is
-       removed and appended to the variant field in the result locale (without
-       case normalization).
-     - When the languageTag argument contains an extlang subtag, the first such
-       subtag is used as the language, and the primary language subtag and other
-       extlang subtags are ignored.
-     - Case is normalized except for variant tags, which are left unchanged.
-       Language is normalized to lower case, script to title case, country to
-       upper case, and extensions to lower case.
-     - If, after processing, the locale would exactly match either ja_JP_JP or
-       th_TH_TH with no extensions, the appropriate extensions are added.
+    ## A locale representing the UK.
 
-     This implements the 'Language-Tag' production of BCP47, and so supports
-     grandfathered (regular and irregular) as well as private use language tags.
+       > Example
+         Get the british locale.
 
-   > Example
-     Creating the locale en_US.
+             import Standard.Base.Data.Locale
 
-         import Standard.Base.Data.Locale
+             example_locale = Locale.uk
+    uk : Locale
+    uk = Locale.from_language_tag "en-GB"
 
-         example_from_tag = Locale.from_language_tag "en_US"
-from_language_tag : Text -> Locale
-from_language_tag tag =
-    java_locale = JavaLocale.forLanguageTag tag
-    from_java java_locale
+    ## A locale representing the United States.
 
-# TODO Dubious constructor export
-from project.Data.Locale.Locale import all
-from project.Data.Locale.Locale export all
+       > Example
+         Get the US locale.
 
-## A type representing a locale.
+             import Standard.Base.Data.Locale
 
-   A locale consists of three parts:
-   - A language, which is mandatory.
-   - A country code, which is optional.
-   - A variant, which is optional.
-type Locale
+             example_locale = Locale.us
+    us : Locale
+    us = Locale.from_language_tag "en-US"
```

Comment:
Again, not directly related and probably low priority but I was always wondering that we are missing the `polish` locale - which seems it could be nice to add it given where the company is based :) What do you think?